packages/react-docs/src/pages/components/modal.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import ComponentDocs from '../../components/componentDocs';
+import PropTypes from 'prop-types';
+import { Modal, Button } from '@patternfly/react-core';
+import Example from '../../components/example';
+
+class ModalDocs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false,
+      showLargeModal: false
+    };
+
+    this.hideModal = this.hideModal.bind(this);
+    this.showModal = this.showModal.bind(this);
+    this.hideLargeModal = this.hideLargeModal.bind(this);
+    this.showLargeModal = this.showLargeModal.bind(this);
+  }
+
+  showModal() {
+    this.setState({
+      showModal: true
+    });
+  }
+
+  hideModal() {
+    this.setState({
+      showModal: false
+    });
+  }
+
+  showLargeModal() {
+    this.setState({
+      showLargeModal: true
+    });
+  }
+
+  hideLargeModal() {
+    this.setState({
+      showLargeModal: false
+    });
+  }
+
+  render() {
+    const { data } = this.props;
+    return (
+      <ComponentDocs data={data}>
+        <Example title="Simple Modal Box">
+          <Button
+            variant="primary"
+            label="Simple Modal Example"
```

Comment:
No need for the label here since it has actual text.  The label (soon to be ariaLabel) is only needed for action buttons that use icons instead of test.